### ISSUE TITLE
Doesn't seem to scale during "Processing : Inspect the media package" workflows

### ISSUE THREAD

**Issue by djcp** [harvard-dce/ec2-management#18]
I uploaded 10 large epiphan videos and it's processing every workflow on two workers. The logs look like:

```Oct  2 16:38:03 monitoring-master1 info: 2015-10-02 16:38:03,998 - INFO - djcp horizontal scaling - Cluster status summary: instances: 10, instances online: 4, workers: 8, workers online: 2, queued_jobs: 0, queued_high_load_jobs: 0, running_jobs: 30
Oct  2 16:38:04 monitoring-master1 info: 2015-10-02 16:38:04,808 - INFO - djcp horizontal scaling - Action summary: stopped 0 instances: ; started 0 instances: 
```

The stats page says admin is running 20 jobs while the two workers are running 4 and 5 jobs respectively. Maybe this is a limitation in matterhorn, but it'd be awesome to spread these puppies out so videos are ready to trim more quickly.

**Comment by djcp** [comment]
![screenshot-2015-10-02t16 44 55](https://cloud.githubusercontent.com/assets/3966/10257346/c963f77a-6924-11e5-9c88-8f85ac1ce2b8.png)
![screenshot-2015-10-02t16 44 30](https://cloud.githubusercontent.com/assets/3966/10257347/c9667bb2-6924-11e5-9fe8-a16efba725b8.png)

**Comment by lbjay** [comment]
Yeah looks like I need to look more at the "queued" job calculation. Of
course I picked this weekend to forget my laptop at work. If I figure out a
way to investigate before Monday I'll check it out.
